tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
-    "compilerOptions": {
-      "outDir": "./build",
-      "lib": [ "es5", "es6" ],
-      "module": "commonjs",
-      "target": "es6",
-      "sourceMap": true
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "src/*"
+      ]
```

Comment:
`"models/*": ["src/models/*"]`

Reply:
Fixed by switching to Jest and specifying in the config file.